src/main/kotlin/com/ecwid/apiclient/v3/dto/webhook/Webhook.kt

```diff
@@ -130,6 +130,15 @@ data class Webhook(
 
 		@SerializedName("promotion.updated")
 		PROMOTION_UPDATED,
+
+		@SerializedName("customer_group.created")
+		CUSTOMER_GROUP_CREATED,
```

Comment:
nit. Let's put it near to `CUSTOMER_` values. The enum looks like it initially had values sorted alphabetically.

Reply:
fixed